llvm/tools/llvm-profgen/PerfReader.cpp

```diff
@@ -60,6 +60,13 @@ static cl::opt<int> CSProfMaxUnsymbolizedCtxDepth(
              "means no depth limit."),
     cl::cat(ProfGenCategory));
 
+static cl::opt<bool> MultiProcessProfile(
+    "multi-process-profile",
```

Comment:
Why do we need a separate cmdline arg to flag multiple pid? Can’t we already know this when user passes multiple pids?

Reply:
See above: this is opt-in for compatibility. We can't use the PID filter argument for this, since it is optional and all PIDs are accepted if it is not passed.